src/audiences/ui/components/rule.js

```diff
@@ -118,6 +126,10 @@ const RuleInput = props => {
 										name={ name }
 										type="text"
 										value={ value }
+										onBlur={ () => {
+											// If the value is empty or in the field data convert back to dropdown on blur.
+											setShowDropdown( value === '' || data.indexOf( value ) >= 0 );
```

Comment:
Is there no way to prevent this duplication of the code on line 86?

Reply:
Good shout, I've set it as a const and reused it everywhere the logic was duplicated.